Remove references to bitwise ops in wasm memory page

pufferfish101007 opened the pull request:
<!-- 👀 Thanks for opening a PR! Read comments like this one to get your PR merged faster. -->
#### Summary
<!-- ✍️ In a sentence or two, describe your changes -->
On https://developer.mozilla.org/en-US/docs/WebAssembly/Reference/Memory:
- Changes main content from `webassembly bitwise instructions` to `webassembly memory instructions`
- Removes note about use of bitwise instructions on floats
- Changes title from `Webassembly memory` to `Webassembly memory instructions` to avoid confusion with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Memory

#### Motivation
<!-- ❓ Why are you making this change? Help us understand how your changes help readers. -->
- the subheading is not what the content is about
- the note is not related to the page's content
- avoids confusion with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Memory (thanks to @OnkarRuikar for pointing this out)

#### Supporting details
<!-- 🔗 Link to supporting information, such as bug trackers, source control, release notes, and vendor docs. -->
n/a

#### Related issues
<!-- 🔨 If applicable, use "Fixes #XYZ" -->
n/a

#### Metadata
<!-- ✅ Check a box if applicable, like this: [x]

This PR…
-->
- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

<!-- 👷‍♀️ After submitting, review the results of the "Checks" tab! -->

pufferfish101007 committed:
Remove references to bitwise ops in wasm memory page

- Changes main content from `webassembly bitwise instructions` to `webassembly memory instructions`
- Removes note about use of bitwise instructions on floats

OnkarRuikar commented:
The real page for the `Memory` object is https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Memory.

I think this one is for memory instructions and should be renamed to `WebAssembly memory instructions` like the [numeric instructions](https://developer.mozilla.org/en-US/docs/WebAssembly/Reference/Numeric) page.

pufferfish101007 committed:
put 'instructions' in title

Title: `Webassembly memory` --> `Webassembly mrmory instructions`.
Avoids confusion with https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/WebAssembly/Memory
(thanks @OnkarRuikar for pointing this out)

dipikabh reviewed (approved):
Thank you for this update!
